### PR TITLE
Changed to using global scope if the provided one is not defined

### PR DESCRIPTION
**Pull request by SergeyRuzin**
Currently if opened URL doesn't match the base one and we try to access items in scopes inside the global one, we get an exception. This causes problems with constantly changing base URL (for example when it contains various request parameters). Now if provided scope is not found, framework will search in global one and write this fact into logs

### CODE REVIEW

**Review by ILastovka — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+[1.8.1] - 2022-01-27
+### Added
```
Comment (andrey-brovko) [inline CHANGELOG.md:8]: ### Changed

Comment (SergeyRuzin) [reply CHANGELOG.md]: Changed

**Review by andrey-brovko — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .idea
 bin
 obj
+src/App/node_modules/
```
Comment (andrey-brovko) [inline .gitignore:4]: We don't have such a folder in the repo.

Comment (SergeyRuzin) [reply .gitignore]: Was a leftover from old app in my folder. Reverted

**Pull request merged**